src/goto-symex/goto_symex.h

```diff
@@ -128,20 +157,20 @@ class goto_symext
   bool constant_propagation;
 
   optionst options;
-  symbol_tablet &new_symbol_table;
 
   /// language_mode: ID_java, ID_C or another language identifier
   /// if we know the source language in use, irep_idt() otherwise.
   irep_idt language_mode;
 
 protected:
-  const namespacet &ns;
-  symex_targett &target;
+  const symbol_tablet &outer_symbol_table;
+  namespacet ns;
```

Comment:
Add comment indicating why we have a mutable one rather than a const ref as previously? Does this always point to `outer_symbol_table`, or...?